R/utils.R

```diff
@@ -575,7 +575,19 @@ merge_2 <- function(dat_1, dat_2, ...) {
   dat_merged <- merge(dat_1, dat_2, ...)
 }
 
-get_plot_coords <- function(plot) {
+#' Takes a ggplot-plot as input and extracts the range of the x-axis coordinates.
+#'
+#' @keywords internal
+#' @noRd
+#'
+#' @param plot ggplot-plot from which to extract the range of the x-axis coordinates.
+#'
+#' @return Numeric value of the range of the x-axis coordinates of the given plot.
```

Comment:
Please write plot in ``
This will highlight the word as R-Syntax in case the documentation is rendered, so it becomes clear it is a function argument.